.github/workflows/release.yml

```diff
@@ -0,0 +1,57 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  linux-x86_64-static:
+    name: Release binary for Linux x86_64
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+    - uses: gmiam/rust-musl-action@master
+      with:
+        args: cargo build --target x86_64-unknown-linux-musl --release --locked
+    - name: Upload icx binaries to release
+      uses: svenstaro/upload-release-action@v2
+      with:
+        repo_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Are there any special precautions we need to take around this secret (like we had with secrets.IC_REF_TOKEN)?